<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>754</width>
    <height>1053</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableView" name="queryView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>711</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>940</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>711</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="title">
    <string>Selection</string>
   </property>
   <widget class="QLineEdit" name="queryLineEdit">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>541</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="queryLabel">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Query Text 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="queryLabel2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>80</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Query Text 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="queryComboLabel">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>140</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Query Combo 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="queryLineEdit2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>80</y>
      <width>541</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="queryComboBox">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>140</y>
      <width>291</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="queryComboLabel2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>192</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Query Combo 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="queryComboLabel3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>245</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
    <property name="text">
     <string>Query Combo 3</string>
    </property>
   </widget>
   <widget class="QComboBox" name="queryComboBox2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>192</y>
      <width>291</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="queryComboBox3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>246</y>
      <width>291</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <italic>false</italic>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
